bin/setup_new

```diff
@@ -80,24 +87,16 @@ has_ruby_version (){
 }
 
 install_ruby_version (){
-  verify_homebrew
```

Comment:
I got rid of the homebrew/darwin piece, it seemed like it wasn't really necessary